pkg/webhook/fleetresourcehandler/fleetresourcehandler_webhook.go

```diff
@@ -136,7 +143,30 @@ func (v *fleetResourceValidator) handleRoleBinding(req admission.Request) admiss
 	if err := v.decodeRequestObject(req, &rb); err != nil {
 		return admission.Errored(http.StatusBadRequest, err)
 	}
-	return validation.ValidateUserForFleetResource(rb.Kind, types.NamespacedName{Name: rb.Name, Namespace: rb.Namespace}, v.whiteListedUsers, req.UserInfo)
+	return validation.ValidateUserForResource(rb.Kind, types.NamespacedName{Name: rb.Name, Namespace: rb.Namespace}, v.whiteListedUsers, req.UserInfo)
+}
+
+func (v *fleetResourceValidator) handleNamespace(req admission.Request) admission.Response {
+	var currentNS corev1.Namespace
+	if err := v.decodeRequestObject(req, &currentNS); err != nil {
+		return admission.Errored(http.StatusBadRequest, err)
+	}
+	fleetMatchResult := regexp.MustCompile(fleetMatch).FindStringSubmatch(currentNS.Name)
```

Comment:
Hi Arvind! It's just a minor thing, but I am wondering if it would be simpler to just use `strings.HasPrefix` for this one as opposed to regexps. Not that there's anything wrong with it though, it's just that for this specific case strings is more lightweight.

Reply:
Same for the `kubematch` below.